### DISCUSSION TITLE
PolygonLayer is missing when zoom in

### DISCUSSION THREAD

**Discussion by sunyonglincanada** [visgl/deck.gl#5344] [Q&A] [answered]
Hi there,

I've tried to use PolygonLayer but found the layer is missing with the negative value for z in the data when zooming in. It works well with non-negative value for z.

Here is the codepen: https://codepen.io/sunyonglincanada/pen/WNGLKqg

Can you please take a look? Thanks!!

**Comment by Pessimistress** [comment, marked as answer]
The default far clipping plane of MapView is placed on the ground. You need to adjust [farZMultiplier](https://deck.gl/docs/api-reference/core/web-mercator-viewport#constructor-1) when working with negative altitude. See example here: https://github.com/visgl/deck.gl/blob/8.3-release/examples/website/data-filter/app.js#L20

**Reply by sunyonglincanada** [reply]
> The default far clipping plane of MapView is placed on the ground. You need to adjust [farZMultiplier](https://deck.gl/docs/api-reference/core/web-mercator-viewport#constructor-1) when working with negative altitude. See example here: https://github.com/visgl/deck.gl/blob/8.3-release/examples/website/data-filter/app.js#L20

Hey, Thanks for your updates. I will try and adjust it based on the example you provided.
